server/index.js

```diff
@@ -57,7 +107,37 @@ app.post('/api/upload', function(req, res) {
 		res.send('Sorry please select the file to upload');
 	}
 });
-
+app.get('/register', function(req, res) {
+	res.render('pages/signup');
+});
+app.post('/register', function(req, res) {
+	const newUser = new User({
+		firstName: req.body.firstName,
+		lastName: req.body.lastName,
+		email: req.body.email,
+		password: req.body.password
+	});
+	bcrypt.genSalt(10, (error, salt) => {
+		bcrypt.hash(newUser.password, salt, (err, hash) => {
+			newUser.password = hash;
+			newUser
+				.save()
```

Comment:
Before saving the account here, you should verify it once in the backend too. Add a regex check for email, and checks for the password too. Also, intimate the user with any errors that come forward.

Reply:
Woking on it